subgraphs/aave-forks/src/constants.ts

```diff
@@ -103,6 +93,7 @@ export const BIGINT_ONE = BigInt.fromI32(1);
 export const BIGINT_THREE = BigInt.fromI32(3);
 export const BIGINT_TEN = BigInt.fromI32(10);
 export const BIGINT_TEN_TO_EIGHTEENTH = BigInt.fromString("10").pow(18);
+export const BIGINT_ONE_RAY = BigInt.fromString("10").pow(27);
```

Comment:
this should be 1^27, that is why we are getting neg rev in some pools

Reply:
isn't 1^27 = 1? :grin:

Reply:
@tnkrxyz any word on this?